src/lib/stitching/exchange/stitching.ts

```diff
@@ -351,6 +352,90 @@ export const exchangeStitchingEnvironment = ({
           },
         },
       },
+      Mutation: {
+        // monkey-patch the existing exchange mutation
+        // to add operations dependent on impulse
+        commerceCreateInquiryOfferOrderWithArtwork: {
+          resolve: async (_source, args, context, info) => {
+            const {
+              conversationLoader,
+              conversationCreateConversationOrderLoader,
+            } = context
+            const {
+              input: { impulseConversationId },
+            } = args
+
+            try {
+              await conversationLoader(impulseConversationId)
+            } catch (e) {
+              // be more elegant?
+              throw new GraphQLError(`Bad Request`)
+            }
+
+            const offerResult = await info.mergeInfo.delegateToSchema({
+              schema: exchangeSchema,
+              operation: "mutation",
+              fieldName: "commerceCreateInquiryOfferOrderWithArtwork",
+              args,
+              context,
+              info,
+              // transforms: [
+              //   new WrapQuery(
+              //     [
+              //       "createInquiryOfferOrderWithArtwork",
+              //       "orderOrError",
+              //       "order",
+              //     ],
+              //     (selectionSet: SelectionSetNode) => {
+              //       // const newSelections = [
+              //       // ...selectionSet.selections,
+              //       return {
+              //         kind: Kind.FIELD,
+              //         name: {
+              //           kind: Kind.NAME,
+              //           value: "internalID",
+              //         },
+              //         selectionSet,
+              //       }
+              //       // ]
+              //       // return { ...selectionSet, selections: newSelections }
+              //     },
+              //     (result) => result
+              //     // result.createInquiryOfferOrderWithArtwork.orderOrError.order
```

Comment:
This is an attempt to add `internalID` to the `order` object in our delegated request using `WrapQuery` but it doesn't seem to have worked (no `internalID` in the delegated exchange request) (example drawn from [here](https://github.com/ardatan/graphql-tools/issues/786#issuecomment-449364838) with a few variations in how we apply the transform, still-unexamined source code from our version of graphql-tools [here](https://github.com/ardatan/graphql-tools/blob/v4.0.5/src/transforms/WrapQuery.ts)).

Reply:
Fixed!